hooks/basic/publish.py

```diff
@@ -252,38 +252,47 @@ def validate(self, task_settings, item):
                 }
             )
 
-        # ---- ensure the published file(s) don't already exist on disk
-
-        conflict_info = None
         if item.properties.get("is_sequence"):
-            seq_pattern = publisher.util.get_path_for_frame(item.properties.publish_path, "*")
-            seq_files = [f for f in glob.iglob(seq_pattern) if os.path.isfile(f)]
-
-            if seq_files:
-                conflict_info = (
-                    "The following files already exist!<br>"
-                    "<pre>%s</pre>" % (pprint.pformat(seq_files),)
-                )
+            path = publisher.util.get_path_for_frame(item.properties.path, "*")
+            publish_path = publisher.util.get_path_for_frame(item.properties.publish_path, "*")
```

Comment:
Is this necessary? Shouldn't those properties already have the correct formatting for them to match?

Reply:
In case of sequence, **item.properties.path** will have the frame number whereas **item.properties.publish_path** will have %4d. 
eg:  /dd/shows/.../RND_999999_td_sgtk_test-sg_geometry_out.v003**.0001**.bgeo.sc
/dd/shows/.../RND_999999_td_sgtk_test-sg_geometry_out.v003.**%4d**.bgeo.sc

Matching the file name pattern using get_path_for_frame().